flixel/input/gamepad/OUYAButtonID.hx

```diff
@@ -16,8 +16,8 @@ class OUYAButtonID
 	public static inline var A:Int = 1;
 	public static inline var LB:Int = 6;
 	public static inline var RB:Int = 7;
-	public static inline var LEFT_ANALOG:Int = 10;
-	public static inline var RIGHT_ANALOG:Int = 11;
+	public static inline var LEFT_STICK_BTN:Int = 10;
```

Comment:
Again, not a fan of the abbreviated `BTN` here - you don't really find that sort of naming in the flixel core source.